src/axom/primal/tests/primal_quadrilateral.cpp

```diff
@@ -0,0 +1,45 @@
+// Copyright (c) 2017-2023, Lawrence Livermore National Security, LLC and
+// other Axom Project Developers. See the top-level LICENSE file for details.
+//
+// SPDX-License-Identifier: (BSD-3-Clause)
+
+#include "axom/config.hpp"
+#include "axom/slic.hpp"
+#include "axom/primal.hpp"
+
+#include "gtest/gtest.h"
+
+namespace primal = axom::primal;
+
+TEST(primal_quadrilateral, area_2D)
+{
+  constexpr int DIM = 2;
+  constexpr double EPS = 1e-12;
+  using CoordinateType = double;
+  using PointType = primal::Point<CoordinateType, DIM>;
+  using QuadrilateralType = primal::Quadrilateral<CoordinateType, DIM>;
+
+  // This is a concave quadrilateral
+  QuadrilateralType quad {PointType {-1.0, 0.1},
+                          PointType {-0.1, 0.5},
+                          PointType {2.0, 0.0},
+                          PointType {0.0, 1.0}};
+
+  const CoordinateType signedArea = quad.signedArea();
+  const CoordinateType area = quad.area();
+
+  EXPECT_NEAR(0.755, signedArea, EPS);
+  EXPECT_NEAR(0.755, area, EPS);
+}
+
+//----------------------------------------------------------------------
```

Comment:
Could you please also add a test for `Quadrilateral<double,3>`?

Reply:
I have not implemented an area calculation for a 3D quadrilateral since it could be skewed in 3D. I could add a test for computing the bounding box from a 3D quadrilateral.